matlab/lib/+Ice/InputStream.m

```diff
@@ -635,7 +635,7 @@ function skipOptional(obj, format)
                     end
                     obj.skip(sz);
                 case Ice.OptionalFormat.Class
-                    obj.readValue([], '');
+                    throw(Ice.MarshalException('', '', sprintf('cannot skip an optional class')));
```

Comment:
Seems like `sprintf` is not required here, as we don't have format params

```suggestion
                    throw(Ice.MarshalException('', '', 'cannot skip an optional class'));
```

Reply:
Yep, good catch!